setup.py

```diff
@@ -61,6 +61,13 @@
 PYTHON_BINDINGS_PATH = os.path.dirname(os.path.abspath(__file__))
 
 
+def set_pkg_config_path():
+    if not pkgconfig.exists("libprecice"):
```

Comment:
pkconfig documentation says that it'll search for libs **where pkg-config was installed**. This will be system directories if pkg-config was installed from distro repos (so most cases). If precice was compiled by the user and installed in a non-system location, `pkgconfig.exists("libprecice")` will be `False` with the default settings.